test/functional/bip65-cltv-p2p.py

```diff
@@ -58,119 +66,79 @@ def create_transaction(self, node, coinbase, to_address, amount):
         tx.deserialize(f)
         return tx
 
-    def get_tests(self):
+    def run_test(self):
+        node0 = BaseNode()
```

Comment:
`self.nodes[0]` is used throughout this test. Consider adding an alias:

```py
node = self.nodes[0]
```
and then using `node` throughout the test.

I think that looks slightly neater for tests that only use one node, but that's just personal preference.